about/roles.md

```diff
@@ -84,12 +84,16 @@ Should a coach not be available to help with an issue, students can also reach
 out to remote coaches (see below). Coaches can also be mentors, which is
 explained below.
 
-<h3 id="mentor">Mentor</h3>
+<h3 id="mentor">Mentor / Project Maintainer</h3>
 
 Mentors are experts on the Open Source project the team is working on. Ideally
 they are decision makers, or work closely with decision makers, so they can set
 general goals, give directions and provide feedback.
 
+At the beginning of summer Mentors will help students design a roadmap for the 
```

Comment:
could you remove “will” here? I think it's best if the whole page is in the present tense. 
Additionally, could you split this sentence into two? Something like:
“At the beginning of summer, mentors help students [...]. They are available throughout the summer to review pull requests, help with concepts, [...].”